<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#d5dfe7;</string>
  </property>
  <widget class="QPushButton" name="requestB">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #C7D2E9;
color:#086A87;
font-weight:bold;
font-size:12px;
border-width: 1px;
border-color:#8BA9E9;
border-style:solid;
border-radius:10px;
}
QPushButton:hover{
background-color: #C7D2E9;
border-style:none;
border-radius:10px;
font-size:18px;
}</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelB">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #C7D2E9;
color:#086A87;
font-weight:bold;
font-size:12px;
border-width: 1px;
border-color:#8BA9E9;
border-style:solid;
border-radius:10px;
}
QPushButton:hover{
background-color: #C7D2E9;
border-style:none;
border-radius:10px;
font-size:18px;
}</string>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:double;
border-left:none;
border-right:none;
border-top:none;
border-color:#2a4b98;
border-width:3px;
color:#0B3861;
padding-top:15px;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:20pt; font-weight:600;&quot;&gt;반납 도서명&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>341</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#d5dfe7;
border-style:none;
color:#0B3861;
text-align:center;
font-size:20px;

</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
